third_party/parquet/parquet_types.cpp

```diff
@@ -3827,11 +3827,11 @@ void ColumnMetaData::__set_type(const Type::type val) {
   this->type = val;
 }
 
-void ColumnMetaData::__set_encodings(const std::vector<Encoding::type> & val) {
```

Comment:
Also here, I would probably do conversion at the API border, since this I am not sure it's actually ok.

Reply:
I think this should be okay, we only use it internally, it can stay std::vector to avoid having to touch all the parquet code